src/services/teams/index.js

```diff
@@ -300,6 +300,10 @@ class Add {
 			importHash,
 			// eslint-disable-next-line no-underscore-dangle
 		} = await this._collectUserAndLinkData({ email, role, teamId });
+		// invite into teams per email should not work for registered students
+		if (user.roles[0].name === 'student') {
+			throw new BadRequest('Invited user is a student');
```

Comment:
I guess if no information about any validation of the email should be handed out it should remain a "BadRequest" but I will change the given error message

Reply:
Reason is that I think giving a "NotFound" implies at least the information that there is no user at all with that email. But giving a BadRequest does not specify the information, what exactly is wrong with the given email.

Reply:
The idea is, that as an attacker, even with knowledge about the code, I can not distinguish between users that dont exist, and users that exist but I am not allowed to know about

If I get a BadRequest instead of NotFound if the user exists but I am not supposed to know about them, I can simply try different emails and thus gain information about what emails are registered in the schulcloud.

Reply:
for example, I can check emails from real people I know about, or emails belonging to hacked accounts from other platforms...